workers/ohsome_quality_analyst/cli.py

```diff
@@ -227,9 +239,23 @@ def create_report(
                 force=force,
             )
         )
-        # TODO: Save as GeoJSON instead of printing to Stdout
+        if outfile:
+            if fid_field is None:
+                fid_field = DATASETS[dataset_name]["default"]
+            feature_collection = asyncio.run(
+                db_client.get_bpolys_from_db(dataset_name, feature_id, fid_field)
+            )
+            for feature in feature_collection.features:
+                feature["properties"].update(vars(report.metadata))
+                feature["properties"].update(vars(report.result))
         click.echo(report.metadata)
         click.echo(report.result)
+    if outfile:
+        """Writing the output GeoJSON"""
+        outfile = pathlib.Path(outfile)
+        outfile.parent.mkdir(parents=True, exist_ok=True)
+        with open(outfile, "w") as f:
+            geojson.dump(feature_collection, f, default=datetime_to_isostring_timestamp)
```

Comment:
This is done for indicators and reports. should it be moved to the [utils/helper](https://github.com/GIScience/ohsome-quality-analyst/blob/main/workers/ohsome_quality_analyst/utils/helper.py) script as a function and called at both instances?

Reply:
yep putting into a helper function would be nice.

Reply:
okay, will do that!